src/Qwiq.Core/IdentityDescriptor.cs

```diff
@@ -1,13 +1,13 @@
 using System;
-using JetBrains.Annotations;
+
```

Comment:
This is great. Do this everywhere. We should no longer have any Jetbrains annotations after migrating. We should instead be using things like nullable annotations